packages/gestalt/src/Link.js

```diff
@@ -14,26 +20,83 @@ import useFocusVisible from './useFocusVisible.js';
 import layoutStyles from './Layout.css';
 
 type Props = {|
+  /**
+   * Supply a short, descriptive label for screen-readers to replace link texts that don't provide sufficient context about the link component behavior. Texts like `Click Here,` or `Read More` can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text to replace the link text. It populates aria-label. Screen readers read the `accessibilityLabel` prop, if present, instead of the link text. See the [Accessibility](https://gestalt.pinterest.systems/link#accessibility) guidelines for more information.
+   */
   accessibilityLabel?: string,
+  /**
+   * Use accessibilitySelected and role when using it as a Tab. See the [Accessibility](https://gestalt.pinterest.systems/link#Accessible-Tab-Link) guidelines for more information.
+   */
   accessibilitySelected?: boolean,
+  /**
+   * Link is a wrapper around components (or children), most commonly [Text](https://gestalt.pinterest.systems/text), so that they become hiperlinks. See the [Link and Text](https://gestalt.pinterest.systems/link#Link-and-Text) variant for more information.
```

Comment:
typo: "hyperlinks"

This wording is also a bit confusing, given that we recommend that Text wraps around Link, not Link wrapping around Text. Perhaps it would be more clear if we refer to "text" — the generic, not our component.

Reply:
DONE